lib/compile-typescript.js

```diff
@@ -13,8 +13,15 @@ module.exports = function compileTypescript(tsconfigFile, projectPath) {
     include: ['lib.*.d.ts']
   });
 
-  let ts = funnel(mergeTrees([libs, projectPath]), {
-    annotation: 'raw source'
+  let src = funnel(projectPath, {
```

Comment:
@dgeb I believe the watching is based on the input node and is handled by Broccoli itself. The `include` arguments don't affect watching because they are plugin-specific.